immersion/kubeflow_pipelines/pipelines/solutions/lab-02_vertex.ipynb

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    PROJECT_ID = !(gcloud config get-value core/project)
</pre><p>nit: you can just <code>get-value project</code>  for brevity</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Nice! Done!</p>